README.md

```diff
@@ -233,13 +231,13 @@ $ npm run component-gen
 
 ## ➤ How to release
 
-We are using [lerna to publish](https://github.com/lerna/lerna/tree/master/commands/publish#readme) our elements
+We are using [lerna to publish](https://github.com/lerna/lerna/tree/main/commands/version#readme) our elements
 
 - Create a branch for releasing。 `git branch release`
 - `git push release` and set the upstream.
-- `npm config delete @tradeshift:registry` delete the registry config so release could publish to different registries
-- `npm run release`
-- `npm config set @tradeshift:registry https://npm.pkg.github.com/` Set back npm to use github for @tradeshift packages
+- `npm run lerna-version` calculate a new version number, generate CHANGELOG.md for all components and make a commit with a predefined commit message.
```

Comment:
Can we have a script to do these steps? 🤔

Reply:
I created a simple shell script, because there are not too much to do, just create a branch and push a commit.

Reply:
It got even better 👍 Thanks for adding it. :)